_data/create.yml

```diff
@@ -0,0 +1,204 @@
+
+- name: "maxParticipants"
+  required: false
+  type: "Number"
+  description: "Set the maximum number of users allowed to joined the conference at the same time."
+
+- name: "logoutURL"
+  required: false
+  type: "String"
+  description: "The URL that the BigBlueButton client will go to after users click the OK button on the ‘You have been logged out message’.  This overrides the value for <code class=\"language-plaintext highlighter-rouge\">bigbluebutton.web.logoutURL</code> in <a href=\"https://github.com/bigbluebutton/bigbluebutton/blob/master/bigbluebutton-web/grails-app/conf/bigbluebutton.properties\">bigbluebutton.properties</a>."
+
+- name: "record"
+  required: false
+  type: "Boolean"
+  description: "Setting ‘record=true’ instructs the BigBlueButton server to record the media and events in the session for later playback. The default is false.<br><br> In order for a playback file to be generated, a moderator must click the Start/Stop Recording button at least once during the sesssion; otherwise, in the absence of any recording marks, the record and playback scripts will not generate a playback file. See also the <code class=\"language-plaintext highlighter-rouge\">autoStartRecording</code> and <code class=\"language-plaintext highlighter-rouge\">allowStartStopRecording</code> parameters in <a href=\"https://github.com/bigbluebutton/bigbluebutton/blob/master/bigbluebutton-web/grails-app/conf/bigbluebutton.properties\">bigbluebutton.properties</a>."
+
+- name: "duration"
+  required: false
+  type: "Number"
+  description: "The maximum length (in minutes) for the meeting.<br><br> Normally, the BigBlueButton server will end the meeting when either (a) the last person leaves (it takes a minute or two for the server to clear the meeting from memory) or when the server receives an <a href=\"#end\">end</a> API request with the associated meetingID (everyone is kicked and the meeting is immediately cleared from memory).<br><br> BigBlueButton begins tracking the length of a meeting when it is created.  If duration contains a non-zero value, then when the length of the meeting exceeds the duration value the server will immediately end the meeting (equivalent to receiving an end API request at that moment)."
+
+- name: "isBreakout"
+  required: false
+  type: "Boolean"
+  description: "Must be set to <code class=\"language-plaintext highlighter-rouge\">true</code> to create a breakout room."
+
+- name: "parentMeetingID"
+  required: "(required for breakout room)"
+  type: "String"
+  description: "Must be provided when creating a breakout room, the parent room must be running."
+
+- name: "sequence"
+  required: "(required for breakout room)"
+  type: "Number"
+  description: "The sequence number of the breakout room."
+
+- name: "freeJoin"
+  required: "(only breakout room)"
+  type: "Boolean"
+  description: "If set to true, the client will give the user the choice to choose the breakout rooms he wants to join."
+
```

Comment:
```suggestion

- name: "breakoutRoomsEnabled"
  required: "Optional(Breakout Room)"
  type: "Boolean"
  default: "true"
  description: "If set to false, breakout rooms will be disabled."
  
  - name: "breakoutRoomsPrivateChatEnabled"
  required: "Optional(Breakout Room)"
  type: "Boolean"
  default: "true"
  description: "If set to false, the private chat will be disabled in breakout rooms."

- name: "breakoutRoomsRecord"
  required: "Optional(Breakout Room)"
  type: "Boolean"
  default: "true"
  description: "If set to false, breakout rooms will not be recorded."
```

Reply:
Looks like we need these ones - a wonderful contributor added them recently https://github.com/bigbluebutton/bigbluebutton.github.io/pull/227/commits/3052df38d46f0274b6802808cc0389c9dd9bad6a :smile_cat: